app/views/avoid_postings/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div id="c-avoid-postings">
+  <div id="a-index">
+    <h1>Avoid Postings</h1>
+
+    <%= render "search" %>
+
+    <table class="striped autofit">
+      <thead>
+      <tr>
+        <th></th>
+        <% if CurrentUser.is_staff? %>
+          <th>Creator</th>
+        <% end %>
+        <th>Status</th>
+        <th>Artist</th>
+        <th>Other Names</th>
+        <th>Details</th>
+        <% if CurrentUser.is_staff? %>
+          <th>Staff Notes</th>
+        <% end %>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @avoid_postings.each do |avoid_posting| %>
+        <tr id="avoid-posting-<%= avoid_posting.id %>" data-artist="<%= avoid_posting.artist_name %>">
+          <td>
+            <%= link_to "View", avoid_posting_path(avoid_posting) %>
+          </td>
+          <% if CurrentUser.is_staff? %>
+            <td>
+              <%= link_to_user avoid_posting.creator %>
+            </td>
+          <% end %>
+          <td>
+            <%= avoid_posting.status %>
+          </td>
+          <td>
+            <%= link_to avoid_posting.artist_name, show_or_new_artists_path(name: avoid_posting.artist_name) %>
+          </td>
+          <td>
+            <% if avoid_posting.artist.present? %>
+              <%= link_to_artists(avoid_posting.other_names) %>
+            <% end %>
+          </td>
+          <td><%= avoid_posting.details %></td>
```

Comment:
DText does not work here, is this intentional?

![Screenshot 2024-07-29 120826](https://github.com/user-attachments/assets/f2acc4b1-faec-4751-9d17-731ceba27090)

Reply:
It's intentional on the versions index, but was not intentional here